x/liquidity/spec/README.md

```diff
@@ -1,21 +1,14 @@
-<!--
-order: 0
-title: Liquidity Overview
-parent:
-  title: "liquidity"
--->
+<!-- order: 0 title: Liquidity Overview parent: title: "liquidity" -->
 
-# `liquidity`
+ # `liquidity`
 
 ## Abstract
 
-### Overview
+This document specifies the Liquidity Module of the Cosmos SDK that serves AMM (Automated Market Makers) style decentralized liquidity providing and coin swap functions.
```

Comment:
is the term "liquidity providing" a thing? can we think of another way to describe adding to the liquidity pool?

Reply:
It is very widely used representation in the industry.
There exists other options, such as "liquidity mining", but it is minor option.
I cannot think of better option widely used in the industry.